network-api/env.default

```diff
@@ -16,3 +16,18 @@ BUILD_THROTTLE_SECONDS=900
 SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=
 SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET=
 SOCIAL_AUTH_LOGIN_REDIRECT_URL=http://localhost:5000/soc/complete/google-oauth2/
+CSP_DEFAULT_SRC='self'
```

Comment:
any lines with just the `'self'` directive will need to also be surrounded by double quotes, otherwise the env loader will strip them out